packages/x-date-pickers-pro/src/internals/utils/valueManagers.ts

```diff
@@ -144,12 +144,12 @@ export const getRangeFieldValueManager = <TDate extends PickerValidDate>({
       ...dateRangeSections.endDate,
     ]);
   },
-  getV6InputValueFromSections: (sections, localizedDigits, isRTL) => {
+  getV6InputValueFromSections: (sections, localizedDigits, isRtl) => {
```

Comment:
I unified to capitalization strategy of this variable with what the core is doing